CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.9.2
+
+- feat: Expose `noOfIsolates` parameter on serve extension on `RelicApp` ([#260](https://github.com/serverpod/relic/pull/260))
+- feat: Add `injectAt` extension method on `Router<T>` ([#261](https://github.com/serverpod/relic/pull/261))
+- feat: Allow re-run of `RelicApp.run` after `close` ([#257](https://github.com/serverpod/relic/pull/257))
```

Comment:
There is also a fix for a regression in hot-reload, but since the bug was never released I have not included the fix in the change log.